charts/sysdig/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 This file documents all notable changes to Sysdig Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.15.11
+### Minor changes
+* KSPM Analyzer - fix container missing CPU\Memory limits & requests
+* KSPM components - Merge analyzer and collector deploy flags into 1
+* KSPM Chart components - fix KSPM serviceAccount that is created when KSPM not installed
```

Comment:
where are the changes with the first and the third items?

Reply:
1 - daemonset-node-analyzer.yaml - added resources
3 - serviceaccount-kspm-collector.yaml - added a condition

Reply:
ah I see